src/Orleans.Transactions.Azure/Orleans.Transactions.Azure.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.Transactions.Azure</PackageId>
+    <Title>Microsoft Orleans Transactions on Azure</Title>
+    <Description>Azure Transaction library of Microsoft Orleans used on the server.</Description>
+    <PackageTags>$(PackageTags) Azure Transactions</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Orleans.Transactions.Azure</AssemblyName>
+    <RootNamespace>Orleans.Transactions.Azure</RootNamespace>
+    <OrleansBuildTimeCodeGen>true</OrleansBuildTimeCodeGen>
```

Comment:
This will cause a test fail once rebased with the updated codege, since we're not generating any code for this assembly, please remove it. Later we'll add a warning to the build if codegen is specified, but we did not find anything to generate code for.